src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -712,31 +707,14 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
                 )
             }
         }.toMutableList()
-    }
 
-    private fun listModuleFilesForRider(module: Module, configScopeId: String): MutableList<ClientFileDto> {
-        val filesInContentRoots = mutableSetOf<VirtualFile>()
-        module.project.guessProjectDir()?.children?.forEach { contentRoot ->
-            if (module.isDisposed) {
-                return@forEach
+        if (isRider()) {
+            computeRiderSharedConfiguration(module.project, configScopeId)?.let {
+                clientFiles.add(it)
             }
-            filesInContentRoots.addAll(visitAndAddFiles(contentRoot, module))
         }
 
-        val forcedLanguages = collectContributedLanguages(module, filesInContentRoots)
```

Comment:
Why did we need to change forcedLanguages logic?

Reply:
It was not changed, it's still in `listModuleFiles`

Before, for retrieving list of files, we had a specific case for Rider, and for the rest. Now, we have a single case for everything, and on top of that we add extension point defined in plugin specific directory. Forced languages were duplicated for those 2 previous cases, now it's at the same place